test/dom5_test.ts

```diff
@@ -563,9 +563,9 @@ suite('dom5', function() {
     });
 
     test('nodeWalkAncestors', function() {
-      // doc -> dom-module -> div -> a
-      const anchor = doc.childNodes![1].childNodes![1].childNodes![0]
-          .childNodes![1].childNodes![3];
+      doc = parse5.parse(docText.replace(/template/g, 'div'));
```

Comment:
This seems potentially confusing. Let's just choose a different path in the doc that doesn't have a template.